source/features/more-dropdown.tsx

```diff
@@ -43,11 +43,11 @@ function onlyShowInDropdown(id: string): void {
 		return;
 	}
 
-	tabItem!.closest('.UnderlineNav-item, li')!.remove();
+	(tabItem!.closest('li') ?? tabItem!.closest('.UnderlineNav-item'))!.remove();
```

Comment:
It didn’t work?

Reply:
If you're talking about #4180: it's not that it didn't work, it's that it's not needed because nothing actually broke in the first place.

This change removes the `<li>` instead of leaving it empty, and it'll work should the devs at GitHub change their minds again.